config/locales/en/wizards/claims/export_users_wizard.yml

```
@@ -0,0 +1,47 @@
+en:
+  wizards:
+    claims:
+      export_users_wizard:
+        page_title: Export users
+        caption: Export users
+        download_csv: Download CSV
+
+        # Claim window step
+        claim_window_step:
+          title: Which claim windows do you want to include?
+          caption: Export users
+          options:
+            all: All claim windows
+            specific: A specific claim window
+          continue: Continue
+
+        # Specific claim window step
+        specific_claim_window_step:
+          title: Choose a claim window
+          caption: Export users
+          continue: Continue
+
+        # Activity level step
+        activity_level_step:
+          title: Which users do you want to include?
+          caption: Export users
+          options:
+            all: All users
+            active: Only active users
```

```suggestion
            active: Active users
```

What does "active users" mean? We understand but do our users? Should we add some hint text?